i18n/cs-CZ.yaml

```diff
@@ -7,4 +7,4 @@ photos: |
     =1 {jednu fotku}
```

Comment:
Using `one {}` instead of `=1 {}` will apply to all numbers which should be singular — not sure if that applies here or not since I don't know Czech :)